proto/pdpb.proto

```diff
@@ -73,6 +73,7 @@ enum ErrorType {
     NOT_BOOTSTRAPPED = 2;
     STORE_TOMBSTONE = 3;
     ALREADY_BOOTSTRAPPED = 4;
+    NOT_COMPATIBLE = 5;
```

Comment:
How about `incompatible`?

For now, only `AskBatchSplit` will be refused, but there may be more incompatible requests, should we define an enum or message?